core/node/consistency_checker/src/lib.rs

```diff
@@ -564,7 +564,7 @@ impl ConsistencyChecker {
             };
             let (version, encoded_data) = commitment_bytes.split_at(1);
             anyhow::ensure!(
-                version[0] == SUPPORTED_ENCODING_VERSION,
+                version[0] == EncodingVersion::Supported.value(),
```

Comment:
consistency checker should work with all encoding versions. not only with the latest one